sale_view_adj/views/sale_order_views.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+    <record id="sale_order_form_inherit" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="client_order_ref" position="move" />
+            </xpath>
+            <xpath expr="//field[@name='partner_shipping_id']" position="attributes">
+                <attribute
+                    name="context"
+                >{'default_type': 'delivery', 'show_address': 1}</attribute>
+            </xpath>
```

Comment:
```suggestion
            <xpath expr="//field[@name='partner_shipping_id']" position="attributes">
                <attribute
                    name="context"
                >{'default_type': 'delivery', 'show_address': 1}</attribute>
            </xpath>
            <xpath expr="//field[@name='payment_term_id']" position="after">
                <field name="client_order_ref" />
            </xpath>
```
Please check if there is any usability issue in case we show the filed in two places.